apps/api/src/app/events/e2e/trigger-event.e2e.ts

```diff
@@ -453,8 +455,19 @@ describe(`Trigger event - ${eventTriggerPath} (POST)`, function () {
         }
       );
 
+      expect(body.data).to.have.all.keys('acknowledged', 'status', 'transactionId');
+      expect(body.data.acknowledged).to.equal(true);
+      expect(body.data.status).to.equal('processed');
+      expect(body.data.transaction).to.be.a.string;
+
```

Comment:
Added more validations to this test as it was my guinea pig for validation during the development of these changes.